docker/Dockerfile

```diff
@@ -0,0 +1,244 @@
+FROM debian:stretch AS stage_build
```

Comment:
Why build in stretch and deploy in buster?

Reply:
No reason here, at some moment I've just switched in the buster in stage_deploy as it was requested. Can be synchronize, but as long there is no dynamic linking we can use any distro as build_stage